pkg/clihelper/show.go

```diff
@@ -184,6 +188,51 @@ func MapWalk(mapID int) error {
 		}
 	}
 
+	if mapInfo.Type == constdef.BPF_MAP_TYPE_HASH.Index() {
+		var key, nextKey uint32
+		// Get the first entry
+		err = goebpfmaps.GetFirstMapEntryByID(
+			uintptr(unsafe.Pointer(&key)),
+			mapID)
+		if err != nil {
+			if errors.Is(err, unix.ENOENT) {
+				fmt.Println("No entries found, empty HASH map (pod_state_map?)")
+				return nil
+			}
+			return fmt.Errorf("unable to get first key (HASH): %v", err)
+		}
+
+		for {
+			var val PodState
+			err = goebpfmaps.GetMapEntryByID(
+				uintptr(unsafe.Pointer(&key)),
+				uintptr(unsafe.Pointer(&val)),
+				mapID)
+			if err != nil {
+				return fmt.Errorf("unable to get HASH entry for key=%d: %v", key, err)
+			}
+
+			fmt.Println("Key : ", key)
```

Comment:
shall we first collect it as a map[string]string and then dump it as json? thus the output can be machine parsable